src/commands/completeInlineTasks.test.ts

```diff
@@ -62,8 +62,6 @@ suite('VSCode Extension Command Tests', function () {
 
     // Open and show required files
     await openAndShowFile('helloWorld.ts')
-    await openAndShowFile('environment.ts')
-    await openAndShowFile('main.ts')
```

Comment:
I think these two were here because I have updated how we search for files to be added to context manager. We are now only allowing to add tabs to avoid adding too many files.

So if they are not in the tabs the task for them will not be executed at all.

The reason for allowing including only tabs is often our old 'safeSearch' or something like that could accidentally include 100s of files from various not correctly ignored patterns in the project